test/security-requirements.test.js

```diff
@@ -0,0 +1,311 @@
+const { linterForRule } = require('./utils');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForRule('az-security-requirements');
+  return linter;
+});
+
+test('az-security-requirements should find errors', () => {
+  const oasDoc = {
+    swagger: '2.0',
+    securityDefinitions: {
+      apim_key: {
+        type: 'apiKey',
+        name: 'Ocp-Apim-Subscription-Key',
+        in: 'header',
+      },
+    },
+    security: [
+      {
+        AADToken: ['read', 'write'],
+      },
+    ],
+  };
+  return linter.run(oasDoc).then((results) => {
+    expect(results.length).toBe(1);
+    expect(results[0].path.join('.')).toBe('security.0.AADToken');
```

Comment:
For my understanding... Where do these values come from?

Reply:
It is just the JSON Path to the element that has the error.  Here it is the "AADToken" property of the 0th element of the "security" property at the top level of the document.